src/main/java/com/textmessenger/controller/DialogController.java

```diff
@@ -26,11 +26,13 @@ public DialogController(DialogService dialogService) {
   }
 
   @PostMapping("/user/{id}")
-  public ResponseEntity createDialog(@PathVariable("id") User user, @RequestBody Dialog dialog) {
-    List<User> users = dialog.getUsers();
-    users.add(user);
-    dialog.setUsers(users);
-    dialogService.createDialog(dialog);
+  public ResponseEntity createDialog(@PathVariable("id") Long user, @RequestBody User mainUser) {
+    System.out.println("===========================================================================");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal "===========================================================================" 3 times. ([squid:S1192](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=127&issues=AWU5sjQMW2AOP0CkKi5l&open=AWU5sjQMW2AOP0CkKi5l)

[ISSUE_KEY]: AWU5sjQMW2AOP0CkKi5l